demostat/__init__.py

```diff
@@ -1,2 +1,11 @@
+def __to_version(release):
+    v = release.split("-")
+    return v[0]
+
+def __is_develop(release):
+    return release.endswith("-dev")
+
 name = "demostat"
-version = "0.1.1-dev"
+release = "0.1.1-dev"
```

Comment:
Der [letzte Release auf GitHub](https://github.com/demostat/demostat/releases/latest) ist aktuell doch 0.1.2. Wäre dann nicht `0.1.3-dev` bzw. `0.1.2-dev` korrekt?

Reply:
Prinzipiell schon, wir müssen dafür nur den aktuellen Release in den dev mergen.
Die Änderung der Versionsnummer sollte aber nicht Teil dieses PR sein